vlc-lua53-compat.patch

```diff
@@ -0,0 +1,30 @@
+commit 41caaa08cde60c4fec4bf2e5f9610e2a1b9e6a23
```

Comment:
Lua in shared-modules isn't compiled with -fPIC option and I would need to patch it in order to do that. I talked on IRC about that (probably with you, if I remember correctly :D) and we decided to compile Lua here.

And I took 5.2 version, because VLC doesn't support 5.3. I know that there are patches which make Lua backwards compatible, but does it make sense to put additional efforts is 5.2 is working well and isn't that outdated?

However, I use qt4 from shared-modules.